script.js

```diff
@@ -1,53 +1,614 @@
+    img: "statics/images/Snapshoot_Portfolio3.png",
+    technolgies: ["html", "Ruby on rails", "css", "javascript"],
+    linkToLiveVersion: "#",
+    linkToSource: "#",
+    generalInf: ["FACEBOOK", "Full Stack Dev", 2015],
+  },
+};
+
+const popUpBtn = document.querySelectorAll(".btn");
+console.log(popUpBtn);
+popUpBtn.forEach((btn) => {
+  btn.addEventListener("click", () => {
+    const mediaQuery = window.matchMedia("(min-width: 1200px)");
+    // Check if the media query is true
+    if (mediaQuery.matches) {
+      const container = document.createElement("div");
+      const propContainer = {
+        boxSizing: "border-box",
+        display: "flex",
+        flexDirection: "column",
+        paddingLeft: "16px",
+        gap: "12px",
+        position: "absolute",
+        width: "60vw",
+        height: "100vh",
+        left: "calc(50% - 60vw/2)",
+        background: "#fff",
+        border: "1px solid #dfe1e6",
+        borderRadius: "16px",
+        top: "0",
+        flex: "none",
+        order: "0",
+        flexGrow: "0",
+        overflow: "auto",
+      };
+      Object.assign(container.style, propContainer);
+      const topBar = document.createElement("div");
+      const topBarProp = {
+        width: "inherit",
+        height: "auto",
+        display: "flex",
+        flexDirection: "row",
+        flex: "1",
+        justifyContents: "space-between",
+      };
+      Object.assign(topBar.style, topBarProp);
+      container.appendChild(topBar);
+      const close = document.createElement("span");
+      const closeProperty = {
+        display: "flex",
+        flexDirection: "row",
+        color: "#aaa",
+        clear: "none",
+        fontSize: "28px",
+        fontWeight: "bold",
+      };
+
+      close.addEventListener("mouseover", () => {
+        close.style.cursor = "pointer";
+        close.style.color = "#000";
+        close.style.textDecoration = "none";
+      });
+      Object.assign(close.style, closeProperty);
+      close.innerHTML = "&times;";
+      const title = document.createElement("p");
+
+      const propTitle = {
+        display: "flex",
+        flexDirection: "row",
+        width: "70%",
+        height: "16px",
+        fontFamily: '"Poppins", sans-serif',
+        fontStyle: "normal",
+        fontWeight: "700",
+        fontSize: "32px",
+        lineHeight: "44px",
+        color: "#172b4d",
+        order: "0",
+        margin: "0",
+      };
+      Object.assign(title.style, propTitle);
+      topBar.appendChild(title);
+      topBar.appendChild(close);
+      const modal = document.createElement("div");
+      const propModal = {
+        display: "none",
+        position: "fixed",
+        zIndex: "99",
+        left: "0",
+        top: "0",
+        width: "100%",
+        maxHeight: "100%",
+        backgroundColor: "rgb(0,0,0)",
+        backgroundColor: "rgba(0,0,0,0.4)",
+        padding: "100px",
+      };
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design, Please make sure your design looks the same as the figma design :pray: 

| Your Design |Figma Design |
| :---:   | :---: | 
| **ON DESKTOP**
![Screenshot from 2022-12-21 22-32-55](https://user-images.githubusercontent.com/92303891/209016150-33d5dae6-a59b-4318-a92d-f3828e68735d.png) **ON MOBILE** ![Screenshot from 2022-12-21 22-49-12](https://user-images.githubusercontent.com/92303891/209017600-c4cffd5b-487b-4309-9073-606b87b49962.png)| **ON DESKTOP** ![Screenshot from 2022-12-21 22-38-29](https://user-images.githubusercontent.com/92303891/209016194-6f26224c-af89-402c-af20-bba20ba616d1.png) **ON MOBILE** ![Screenshot from 2022-12-21 22-45-49](https://user-images.githubusercontent.com/92303891/209017439-e4b5e584-f8aa-4ef5-8693-8e733967ec94.png)|

Reply:
Thanks. Tried to reach as close as possible to figma.